identity/handler.go

```diff
@@ -28,6 +28,9 @@ const RouteItem = RouteCollection + "/:id"
 const RouteValidate = RouteCollection + "/validate"
 
 // fandom-end
+const RouteMultifactorDelete = RouteCollection + "/:id/credential/:credentialType"
```

Comment:
I'd change the const name not to contain `Delete`. As I see in other constants here, they don't say anything about HTTP method to be used.

Reply:
Also, it's possible to base it on `RouteItem` instead of `RouteCollection`